tests/schedule/test_train_schedule.py

```diff
@@ -0,0 +1 @@
+# Schedule is testable when trains are implemented.
```

Comment:
I understand this comment. But you could add tests to verify that you can add objects to the database from nested dictionaries. I'm very interested to see if this works out. Do you know what I mean? 😇

Reply:
I will add it as soon as you abstract test cases for this purpose are merged :)